fetch/nosniff/resources/nosniff-first-empty-double.asis

```diff
@@ -0,0 +1,7 @@
+HTTP/1.1 200 YOU HAVE NO POWER HERE
+Content-Length: 22
+Content-Type: x/x
+X-Content-Type-options:
```

Comment:
Is options deliberately lowercase in these tests?  Shouldn't hurt, but as that's not what they're aimed at testing, should probably capitalize it.